src/Dialogs/BulkRenamer/RenamerDialog.vala

```diff
@@ -294,6 +301,9 @@ public class Files.RenamerDialog : Gtk.Dialog {
         }
         renamer.modifier_chain.add (mod);
         var mod_button = new Gtk.Button.with_label (mod.mode.to_string ()) {
+            always_show_image = true,
+            image = new Gtk.Image.from_icon_name ("pan-down-symbolic", Gtk.IconSize.MENU),
```

Comment:
It wasn't immediately obvious to me that these are menu buttons with options, so let's give them a dropdown icon to make that clearer